### PR TITLE
Added styling for sticky header.

### LINKED ISSUE #73: 💡 [Feature]: Make the header sticky
When we scroll, the header scrolls. That's not so intuitive. Let's make it sticky.

Applicable to both demo landing and individual demo pages.

### PR DESCRIPTION
**Pull request by abhikhatri67**
# Description

Added styling for a sticky header for pages. Fixed #73 

Fixes #73 

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Tested styling changes in the browser.

# Checklist:

- [x] My code follows the [style guidelines](https://github.com/atapas/webapis-playground/blob/master/HOW-TO-ADD-DEMO.md) of this project
- [x] I have performed a self-review of my own code
- [x] I have commented on my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

File: styles/globals.css [hunk 1]
```diff
@@ -167,3 +167,9 @@ code {
 .text-red-600 {
   color: #dc2626;
 }
+
+.sticky-header {
```
Comment (atapas) [inline styles/globals.css:171]: @abhikhatri67 Can we handle it in tailwind itself? I have a reservation about putting code in global.css. We may get rid of this file soon.

**Review by atapas — CHANGES_REQUESTED** [review]
@abhikhatri67 A tiny comment to take care

File: styles/globals.css (hunk 1 above)
Comment (abhikhatri67) [reply styles/globals.css:171]: Sure, I will change it.

**Review by atapas — APPROVED** [review]
@abhikhatri67 Changes are looking great. approving

**Comment by atapas** [comment]
@abhikhatri67 

Thanks a lot for taking the issue and making the app better. Please feel free to pick up more, contribute more. I wish you a great #Hactoberfest ahead. You will be added to the contribution list right after we merge this PR. Don't forget to share about your contribution publicly and provide your support with a star ⭐.

Great work 🔥 !!!!

**Comment by atapas** [comment]
@all-contributors please add @abhikhatri67  for Code

**Pull request merged**